src/daft-sql/src/python.rs

```diff
@@ -23,8 +23,16 @@ pub fn sql_expr(sql: &str) -> PyResult<PyExpr> {
 }
 
 #[pyfunction]
-pub fn list_sql_functions() -> Vec<String> {
-    SQL_FUNCTIONS.map.keys().cloned().collect()
+pub fn list_sql_functions() -> Vec<(String, &'static str, Vec<&'static str>)> {
```

Comment:
this return type is kinda hard to reason about. Could we create a struct/pyclass for this instead.